Source/Core/DolphinQt/Achievements/AchievementBox.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef USE_RETRO_ACHIEVEMENTS
+#include <QGroupBox>
+
+#include "Core/AchievementManager.h"
+
+class QLabel;
+class QProgressBar;
+class QWidget;
+
+struct rc_api_achievement_definition_t;
+
+class AchievementBox final : public QGroupBox
+{
+  Q_OBJECT
+public:
+  explicit AchievementBox(QWidget* parent, rc_client_achievement_t* achievement);
+  void UpdateData();
+
+private:
+  QLabel* m_badge;
+  QLabel* m_title;
```

Comment:
It looks like `m_title` doesn't need to be a member variable. It can just be a local variable in the constructor. Same thing for `m_points` and maybe others?